src/main/kotlin/com/angorasix/projects/management/core/presentation/handler/ProjectsManagementHandler.kt

```diff
@@ -116,11 +117,11 @@ class ProjectsManagementHandler(
      * @return the `ServerResponse`
      */
     suspend fun createProjectManagement(request: ServerRequest): ServerResponse {
-        val requestingContributor = request.attributes()[apiConfigs.headers.contributor]
+        val requestingContributor = request.attributes()[AngoraSixInfrastructure.REQUEST_ATTRIBUTE_CONTRIBUTOR_KEY]
         return if (requestingContributor is SimpleContributor) {
             val project = try {
                 request.awaitBody<ProjectManagementDto>()
-                    .convertToDomain()
+                    .convertToDomain(setOf(SimpleContributor(requestingContributor.contributorId, emptySet())))
```

Comment:
😕 not sure why we're not simply doing `.convertToDomain(setOf(requestingContributor))`, I guess because we haven't defined how we want to handle the `SimpleContributor.grants` yet...probably not the best decision, but well, we can fix it across the board later on 👍